src/server.ts

```diff
@@ -105,18 +105,31 @@ export async function restart(context: vscode.ExtensionContext, verificationStat
         return;
     }
 
+    let prustiServerCwd: string | undefined;
+    if (vscode.workspace.workspaceFolders !== undefined) {
+        prustiServerCwd = vscode.workspace.workspaceFolders[0].uri.path;
+        util.log(`Prusti server will be executed in '${prustiServerCwd}'`);
+    }
+
+    const prustiServerArgs = ["--port", "0"].concat(
```

Comment:
Again, we could move the `--port` setting to `extraPrustiServerArgs` instead of hardcoding it?

Reply:
Do you see a use case for letting the user choose the port?

Reply:
Not really, but I don't see much of a reason for the `--port` setting in the first place aha

Reply:
Yeah :D We could at least make 0 the default.

Reply:
Makes sense, in that case we can merge as is